Description: Make doc build reproducible
Author: Chriss Lamb <lamby@debian.org>
Forwarded: https://review.openstack.org/#/c/340354/
Last-Update: 2016-07-11

--- python-openstackclient-2.6.0.orig/doc/source/conf.py
+++ python-openstackclient-2.6.0/doc/source/conf.py
@@ -80,7 +80,7 @@ release = version_info.release_string()
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = ['**tests**']
 
 # The reST default role (used for this markup: `text`) to use for all
 # documents.
